src/main/java/io/fabric8/vertx/maven/plugin/mojos/RunMojo.java

```diff
@@ -33,8 +36,111 @@
     requiresDependencyResolution = ResolutionScope.COMPILE_PLUS_RUNTIME)
 public class RunMojo extends AbstractRunMojo {
 
+    /**
```

Comment:
sorry please ignore my comment, you mean to say providing `args` like what you told applies for vert.x launcher and custom launcher - so user can pass what they want to their launchers.